test/e2e/protocol/proposer.test.mjs

```diff
@@ -14,65 +13,52 @@ chai.use(chaiAsPromised);
 
 const environment = config.ENVIRONMENTS[process.env.ENVIRONMENT] || config.ENVIRONMENTS.localhost;
 
-const { bond, gasCosts, txPerBlock, mnemonics, signingKeys } = config.TEST_OPTIONS;
+const { signingKeys } = config.RESTRICTIONS;
+const { bond, gasCosts, mnemonics, signingKeys: testSigningKeys } = config.TEST_OPTIONS;
 
-const testProposers = [
-  new Nf3(signingKeys.proposer1, environment),
-  new Nf3(signingKeys.proposer2, environment),
-  new Nf3(signingKeys.proposer3, environment),
-];
+const bootProposer = new Nf3(signingKeys.bootProposerKey, environment);
+const testProposer = new Nf3(testSigningKeys.proposer1, environment);
 
 const web3Client = new Web3Client();
 
 describe('Basic Proposer tests', () => {
   before(async () => {
-    for (const prop of testProposers) {
-      await prop.init(mnemonics.proposer);
-    }
-
-    // Proposer registration
-    await testProposers[0].registerProposer();
-    // Proposer listening for incoming events
-    const newGasBlockEmitter = await testProposers[0].startProposer();
-    newGasBlockEmitter.on('gascost', async gasUsed => {
-      logger.debug(
-        `Block proposal gas cost was ${gasUsed}, cost per transaction was ${gasUsed / txPerBlock}`,
-      );
-    });
-    await testProposers[0].addPeer(environment.optimistApiUrl);
+    await testProposer.init(mnemonics.proposer);
+    await bootProposer.init(mnemonics.proposer);
   });
 
-  it('should register a proposer', async () => {
+  it('should register the boot proposer', async () => {
     let proposers;
-    ({ proposers } = await testProposers[1].getProposers());
+    ({ proposers } = await bootProposer.getProposers());
+
     // we have to pay 10 ETH to be registered
-    const startBalance = await web3Client.getBalance(testProposers[1].ethereumAddress);
-    const res = await testProposers[1].registerProposer();
+    const startBalance = await web3Client.getBalance(bootProposer.ethereumAddress);
+    const res = await bootProposer.registerProposer();
```

Comment:
registration of a proposer nos requires a URL

Reply:
same, will be resolved since I'll have to rebase